app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/vaultunlock/VaultUnlockViewModel.kt

```diff
@@ -133,6 +139,17 @@ class VaultUnlockViewModel @Inject constructor(
 
     private fun handleDismissDialog() {
         mutableStateFlow.update { it.copy(dialog = null) }
+        when {
+            state.fido2GetCredentialsRequest != null -> {
+                sendEvent(VaultUnlockEvent.Fido2GetCredentialsError)
```

Comment:
Hmmm, is this supposed to be here?

Reply:
Yes. When the dialog is dismissed and we have a FIDO2 request in state we have to notify Android an error occurred. This event is how we're triggering the appropriate error response.